packages/@monocdk-experiment/assert/package.json

```diff
@@ -46,7 +47,7 @@
     "pkglint": "0.0.0",
     "ts-jest": "^25.2.0",
     "@monocdk-experiment/rewrite-imports": "0.0.0",
-    "monocdk-experiment": "0.0.0",
+    "monocdk-experiment": "file:../../monocdk-experiment",
```

Comment:
I don't understand why this change is there?

Reply:
Will revert. Good point